### ISSUE TITLE
Tensorpack doesn't save batch_norm parameters to model

### ISSUE THREAD

**Issue by chenchen718** [ppwwyyxx/tensorpack#499]
I use `tf.contrib.layers.batch_norm` in my model. When testing, I cannot reuse any parameters such as 'moving_mean', 'moving_variance', 'beta' in batch norm. It complains that these variables do not exist. It seems that when training, tensorpack does not save them to the trained model. The other model parameters are no problem. 

What should I do to make sure that batch_norm parameters are saved?

Thanks very much!

**Comment by ppwwyyxx** [comment]
Those variables will be saved.

By "when testing", did you mean "when you define the graph for testing"?

**Comment by ppwwyyxx** [comment]
In any case, for others to under stand what the problem is, please post "what you've observed", as suggested in the issue template.

**Comment by chenchen718** [comment]
I am sorry. It turned out to be my own carelessness. These parameters are saved fine.  I should remove this issue.  
Thanks!
